multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,25 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl
+    permissions: 0755
+    content: |
+      # source this file to set up a kubectl alias
+      sudo rkt run \
+        --net=host \
+        --volume bk,kind=host,source=/srv/kubernetes \
+        --mount volume=bk,target=/srv/kubernetes \
+        --insecure-options=image docker://{{.HyperkubeImageRepo}}:{{.K8sVer}} \
+        --exec /hyperkube -- kubectl $@
```

Comment:
I don't think so, seeing as we want $@ to expand as an argument array to append to 

`./hyperkube kubectl arg1 arg2 .. argn`

rather than concatenating:

`/hyperkube kubectl "arg1 arg2 .. argn` if we quoted `$@`